pre-docs/applying_manifest_blocks.md

```diff
@@ -168,9 +168,7 @@ Manifest block compilation can access Hiera data that you add to your Bolt confi
 
 Following the Hiera 5 convention, the default data dir is relative to `hiera.yaml` at `$BOLTDIR/data`. For config file examples, see https://puppet.com/docs/puppet/5.5/hiera_config_yaml_5.html.
 
-If a custom data provider is used (such as `hiera-eyaml`, which allows you to encrypt your data) the gem dependencies must be available to Bolt. If using `puppet-bolt` packages and installing `hiera-eyaml`
-- on Windows, run `"C:/Program Files/Puppet Labs/Bolt/bin/gem.bat" install hiera-eyaml`
-- on other platforms, run `/opt/puppetlabs/bolt/bin/gem install hiera-eyaml`
+If a custom data provider is used (such as `hiera-eyaml`, which allows you to encrypt your data) the gem dependencies must be available to Bolt (for example `gem install hiera-eyaml`). See [Installing Gems with Bolt Packages](bolt_installing.md#installing-gems-with-bolt-packages).
```

Comment:
do we really want the example?

Reply:
I can remove it. I guess it's better to point people to the full command only.